examples/simple-druid-cluster.yaml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
I just skimmed the file, but what's the difference to the `derby` variant? If they are the same, i don' think we should keep both.

Reply:
Nothing, didnt want to destroy your structure!